ecl/hql/hqlgram2.cpp

```diff
@@ -12339,7 +12340,7 @@ void parseAttribute(IHqlScope * scope, IFileContents * contents, HqlLookupContex
             const bool isMacro = parsed->isMacro();
             const bool neverSimplify = isMacro || ctx.neverSimplify(fullName);
             OwnedHqlExpr simplified = neverSimplify ? nullptr : createSimplifiedDefinition(parsed);
-            if (ctx.hasCacheLocation() && !ctx.ignoreSimplified())
+            if (ctx.hasCacheLocation() && !ctx.ignoreCache())
```

Comment:
I'm not sure that this is correct - it should mean ignore the contents of the cache, but not prevent it being updated.

Should it be just testing ctx.hasCacheLocation, or is there a reason for checking something else?

Reply:
@ghalliday I thought ignoreCache implies that it doesn't update the cache either but I understand your point that metaCache set to blank does the same thing.  I'll update the code to allow updates when ignoreCache=true